src/styles/index.scss

```diff
@@ -1,3 +1,82 @@
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  border: dotted #000 1vmin;
```

Comment:
According to the Code Style Rules, it's recommended to use HEX color codes instead of color keywords. Please replace `#000` with the equivalent HEX color code.